src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="#">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="#" class="nav__link">Apple</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+            <a href="#" class="nav__link">Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Smartphones</a>
+            <a href="#" class="nav__link">Smartphones</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Laptops & computers</a>
+            <a href="#" class="nav__link">Laptops & computers</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Gadgets</a>
+            <a href="#" class="nav__link">Gadgets</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Tablets</a>
+            <a href="#" class="nav__link">Tablets</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Photo</a>
+            <a href="#" class="nav__link">Photo</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Video</a>
+            <a href="#" class="nav__link">Video</a>
```

Comment:
:harold:
![image](https://user-images.githubusercontent.com/37161315/215842099-181be585-f2d0-4faf-9c8d-cd5d700e6a47.png)

Reply:
I had to add '#' because of lint :{